extensions/sql-bindings/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -53,22 +53,28 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined):
 	}));
 
 	if (!azureFunctionName) {
+		TelemetryReporter.sendErrorEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.getAzureFunctionProject);
 		return;
 	}
+	TelemetryReporter.sendActionEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.getAzureFunctionProject);
 
 	// 2. select input or output binding
 	const selectedBinding = await azureFunctionsUtils.promptForBindingType();
 
 	if (!selectedBinding) {
+		TelemetryReporter.sendErrorEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.getBindingType);
 		return;
 	}
+	TelemetryReporter.createActionEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.getBindingType).withAdditionalProperties({ bindingType: selectedBinding.label }).send();
```

Comment:
Don't use the label for that - we should never be sending up potentially localized strings. Use the type enum value.